src/plugins/login/loginPlugin.ts

```diff
@@ -11,11 +11,16 @@ export class AzureLoginPlugin {
     this.provider = (this.serverless.getProvider('azure') as any) as AzureProvider;
 
     this.hooks = {
-      'before:deploy:initialize': this.login.bind(this)
+      'before:package:initialize': this.login.bind(this)
```

Comment:
Not entirely sure of the difference here, but the warning I got said to use `package:initialize` without the before. Guess it probably doesn't matter though, right?